app/scripts/modules/amazon/src/serverGroup/details/rollback/rollbackServerGroup.html

```diff
@@ -46,6 +61,10 @@
               <strong>max</strong>: {{ serverGroup.capacity.max }},
               <strong>desired</strong>: {{ serverGroup.capacity.desired }}
             ]<br/>(minimum capacity pinned at {{serverGroup.capacity.desired}} to prevent autoscaling down during rollback)
+              <div ng-if="command.rollbackContext.targetHealthyRollbackPercentage < 100">
```

Comment:
If this were also an `<li>`, it might read a little more cleanly, adding a separate step in the list, e.g.
```html
<li ng-if="command.rollbackContext.targetHealthRollbackPercentage < 100">
  Wait for at least {{minHealthy(command.rollbackContext.targetHealthRollbackPercentage)}} instances to report as healthy
</li>
```

Reply:
This does look better, thanks!